ocaml/xapi/cert_distrib.ml

```diff
@@ -0,0 +1,247 @@
+module Unixext = Xapi_stdext_unix.Unixext
+
+module D = Debug.Make (struct let name = "cert_distrib" end)
+
+module XenAPI = Client.Client
+
+type existing_cert_strategy = Erase | Merge [@@deriving sexp]
```

Comment:
It's not immediately clear what this implies on the operation. I was thinking if instead of Erase. something like `Erase_old` or `Only_new` would make more sense. An explanation of the implication on the mli would be welcome as well.